### ISSUE TITLE
[Docs] Add a FAQ page

### ISSUE THREAD

**Issue by wuchong** [alibaba/fluss#103]
### Search before asking

- [X] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.

### Motivation

Currently, there are many users have interests in Fluss, and there are many common questions, such as "what's the difference between Fluss and Kafka?", "what's the difference between Fluss and Paimon?". We can create a FAQ page to address such questions. 

### Solution

Maybe we can add a "Project" dropdown section in the top menu, and move "Community", "Roadmap" under the "Project" dropdown menu. Add the "FAQ" under "Project" as well. 

### Willingness to contribute

- [ ] I'm willing to submit a PR!

**Comment by psxjoy** [comment]
I think we should also align on  code style. Can we provide an XML file for IDEA's coding guidelines?just like [dubbo codestyle](https://github.com/apache/dubbo/tree/3.3/codestyle)

**Comment by wuchong** [comment]
> I think we should also align on  code style. Can we provide an XML file for IDEA's coding guidelines?just like [dubbo codestyle](https://github.com/apache/dubbo/tree/3.3/codestyle)
> 
> 

Yes. We already have such a checkstyle plugin and a spotless to check the code style. See more https://alibaba.github.io/fluss-docs/docs/dev/ide-setup/#required-plugins

**Comment by psxjoy** [comment]
> > I think we should also align on  code style. Can we provide an XML file for IDEA's coding guidelines?just like [dubbo codestyle](https://github.com/apache/dubbo/tree/3.3/codestyle)
> 
> Yes. We already have such a checkstyle plugin and a spotless to check the code style. See more https://alibaba.github.io/fluss-docs/docs/dev/ide-setup/#required-plugins

Thanks for the clarification. I overlooked this part of the webpage, and it indeed meets my expectations.
